entry/client/index.js

```diff
@@ -0,0 +1,11 @@
+// '__sku_alias__clientEntry' is a webpack alias
+// pointing to the consuming apps client entry
+import client from '__sku_alias__clientEntry';
+
+import { loadableReady } from '@loadable/component';
+
+import configKey from '../configKey';
+
+loadableReady(() => {
+  client(window[configKey] || {});
```

Comment:
Should we be deleting the configKey off window after its passed back into client function?